content/guides/embed/ui-elements/theming-styling.md

```diff
@@ -0,0 +1,122 @@
+---
+rank: 11
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/explorer
+  - embed/ui-elements/uploader
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+<!--alex ignore -->
+# Theming & styling for Box UI Elements
+
+With theming & styling for Box UI Elements you can customize the look of the
+embedded Box component to your enterprise's requirements.
+<!--alex ignore -->
+The tool provides a real-time live preview for an easy adjustment of the theme.
+<!--alex enable -->
+
+<!--alex ignore -->
+<Message type='notice'>As of now, theming & styling is available for Content
+Explorer and Content Uploader.
+</Message>
+<!--alex enable -->
+
+## How to start
+
+Add the selected BUIE to your application. The default theme template will be applied.
```

Comment:
I don't think we use term BUIE externally. 

The default Box theme is applied.